packages/bindings/src/lib.rs

```diff
@@ -1,8 +1,16 @@
 mod msg;
+mod querier;
 mod query;
 
 pub use msg::{SwapMsg, TerraMsg};
+pub use querier::TerraQuerier;
 pub use query::{
-    ExchangeRateResponse, ExchangeRatesResponse, SimulateSwapResponse, SwapQuery, TerraQuerier,
-    TerraQuery,
+    ExchangeRateResponse, ExchangeRatesResponse, OracleQuery, RewardsWeightResponse,
+    SeigniorageProceedsResponse, SimulateSwapResponse, SwapQuery, TaxCapResponse,
+    TaxProceedsResponse, TaxRateResponse, TerraQuery, TobinTaxResponse, TreasuryQuery,
 };
+
+// This export is added to all contracts that import this package, signifying that they require
+// "terra" support on the chain they run on.
+#[no_mangle]
+extern "C" fn requires_terra() {}
```

Comment:
Do I understand correctly that the `terra` feature never exists in https://github.com/CosmWasm/cosmwasm but only contract+chain, passed intouched by the VM?

Reply:
It never exists in cosmwasm, only for contracts that import these custom extensions.